src/consumer.rs

```diff
@@ -100,6 +101,7 @@ impl TryFrom<ConsumerConfig> for NativeConsumerConfig {
             }
         }
         let config = builder.build().map_err(|e| format!("{}", e))?;
+        web_sys::console::log_1(&format!("{:#?}", &config).into());
```

Comment:
Do we need to log this?

Reply:
Ah I was doing that to verify that the JS value got converted correctly. We can remove it